api/components/com_users/src/Controller/UsersController.php

```diff
@@ -37,6 +40,23 @@ class UsersController extends ApiController
 	 */
 	protected $default_view = 'users';
 
+	/**
+	 * The default view for the display method.
+	 *
+	 * @var    array
+	 * @since  3.0
+	 */
+	protected $supportedRange = [
+		'past_week',
+		'past_1month',
+		'past_3month',
+		'past_6month',
+		'past_year',
+		'past_year',
```

Comment:
```suggestion
		'post_year',
```
post_year = more than a year ago